modules/tensor_mechanics/test/tests/beam/action/tests

```diff
@@ -1,13 +1,30 @@
 [Tests]
   [./2_block_action]
-    test = Exodiff
+    type = 'Exodiff'
     input = '2_block.i'
     exodiff = '2_block_out.e'
   [../]
   [./2_block_common_action]
-    test = Exodiff
+    type = 'Exodiff'
     input = '2_block_common.i'
     exodiff = '2_block_out.e'
     prereq = '2_block_action'
   [../]
+  [./beam_action_test1]
+    type = 'RunException'
+    input = 'beam_action_chk.i'
+    expect_err = 'LineElementAction: A vector of displacement variable names should be provided as'
```

Comment:
Shouldn't the full error statement be provided here? The last part of this statement is 'input using `displacements`.'

Reply:
I think the expect_err just has to be unique, so you don't have to put the whole thing in there, but if you want to for completeness, go ahead.

Reply:
Thanks, Al. I did not know that.